Modules/include/UserInputValidator.h

```diff
@@ -137,16 +141,23 @@ namespace ChimeraTK {
     bool validateAll();
 
    protected:
+    static constexpr std::string_view tagValidatedVariable{"__UserInputValidator"};
+
+    // Helper function to set up queue lengths of valid values
+    void enableDeepValidation();
```

Comment:
Maybe mention in the comment that this function is called in the first call to validate() automatically.

Also maybe rename it back into finalise()

Reply:
Also fixed